README.rst

```diff
@@ -1339,8 +1341,8 @@ Add ``openwisp_utils.loaders.DependencyLoader`` to ``TEMPLATES`` in your ``setti
             'OPTIONS': {
                 'loaders': [
                     'django.template.loaders.filesystem.Loader',
-                    'django.template.loaders.app_directories.Loader',
                     'openwisp_utils.loaders.DependencyLoader',
+                    'django.template.loaders.app_directories.Loader',
```

Comment:
Changed the order as it was reponsible for failing tests in `openwisp_radius`. The related PR is :point_down: 
https://github.com/openwisp/openwisp-radius/pull/214